### PR TITLE
Bugs/small errors

### PR DESCRIPTION
**Pull request by xkevin190**
message bugs fixed:
- error handling when a plain message arrives
- validations that verify that the message is from a contact and that it is addressed to him
- Errors in status messages

### CODE REVIEW

**Review by jeandudey — APPROVED** [review]

File: __tests__/Views/configuration/AdministrativeComponent-test.js [hunk 1]
```diff
@@ -118,4 +120,30 @@ describe('test Administrative component', () => {
     wrapper.instance().sendDialToChatService('test', () => { });
     expect(mocksendDialfalse.mock.calls.length).toBe(1);
   });
+
```
Comment (luisan00) [inline __tests__/Views/configuration/AdministrativeComponent-test.js:123]:
```suggestion
```

File: __tests__/Views/configuration/AdministrativeComponent-test.js [hunk 2]
```diff
@@ -118,4 +120,30 @@ describe('test Administrative component', () => {
...
+
+  test('check componentDidMount', async () => {
+    const instance = wrapper.instance();
+    await instance.componentDidMount();
+    expect(instance.state.upnp).toBe(true);
+  });
+
```
Comment (luisan00) [inline __tests__/Views/configuration/AdministrativeComponent-test.js:130]:
```suggestion
```

File: __tests__/Views/configuration/AdministrativeComponent-test.js [hunk 3]
```diff
@@ -118,4 +120,30 @@ describe('test Administrative component', () => {
...
+
+  test('activate upnp', () => {
```
Comment (luisan00) [inline __tests__/Views/configuration/AdministrativeComponent-test.js:132]: it's not very important in this point but the correct spelling is UPnP, which stands for **U**niversal **P**lug a**n**d **P**lay 
```suggestion
  test('activate UPnP', () => {
```

File: __tests__/Views/configuration/AdministrativeComponent-test.js [hunk 4]
```diff
@@ -118,4 +120,30 @@ describe('test Administrative component', () => {
...
+    jest.useFakeTimers();
+
+    wrapper.setState({ upnp: false });
+    wrapper.find('Styled(Switch)').at(1).props().onTouchEnd();
+    setTimeout(async () => {
+      await expect(wrapper.instance().state.upnp).toBe(true);
+    }, 500);
+  });
+
+  test('disable upnp', () => {
```
Comment (luisan00) [inline __tests__/Views/configuration/AdministrativeComponent-test.js:142]:
```suggestion
  test('disable UPnP', () => {
```

**Review by luisan00 — CHANGES_REQUESTED** [review]

**Review by luisan00 — APPROVED** [review]

**Pull request merged**